common/source/docs/common-compass-calibration-in-mission-planner.rst

```diff
@@ -87,12 +81,22 @@ If calibration fails:
 - to cancel calibration at anytime hold throttle stick full up and full left yaw for 2 seconds
 - if, after multiple attempts, you are unable to calibrate the compass, Cancel with stick and use normal Onboard Calibration from GCS above
 
+Large Vehicle MagCal
+====================
+
+Large or heavy vehicles are impractical to rotate on all axis. This feature allows a fairly accurate calibration if GPS lock is active on the autopilot and the vehicles actual heading is know, either using a landmark reference on the Mission Planner map, or using another compass (eg cell phone) and entering the vehicles heading.
+
+Compass Ordering
+================
+
+At the top of the page, you can change the priority of the attached compasses, if desired.
+
 Offboard Calibration (aka "Live Calibration")
 =============================================
 
-.. note:: This method is not recommended if you are using a recent version of ArudPilot that supports the Onboard Calibration described above.
+.. note:: This method is not recommended if you are using a recent version of ArudPilot that supports the Onboard Calibration described above. It is no longer available in Mission Planner.
```

Comment:
Maybe we should just zap this whole part if it's gone from MP?  .. and remove any other references to "Live" calibration.  happy days!

Reply:
Fine by me!!gone!